portal/messages/es.json

```diff
@@ -21,14 +21,14 @@
     "insufficient-balance": "{symbol} insuficiente",
     "learn-more": "Aprenda más",
     "mainnet-is-now-live": "¡Mainnet está disponible!",
-    "mainnet-is-now-live-description": "Tunelice, apueste y más en mainnet usando el nuevo cambio de red.",
+    "mainnet-is-now-live-description": "Tunelice, hace stake y más en mainnet usando el nuevo cambio de red.",
```

Comment:
```suggestion
    "mainnet-is-now-live-description": "Tunelice, haga stake y más en mainnet usando el nuevo cambio de red.",
```

Reply:
I think the whole "Mainnet is now live" is no longer part of the code, so I think it can be removed. Can you check if they're being used? See https://github.com/hemilabs/ui-monorepo/pull/1179